java-frontend/src/main/java/org/sonar/java/bytecode/cfg/BytecodeCFGBuilder.java

```diff
@@ -393,7 +393,8 @@ public void visitLabel(Label label) {
 
     @Override
     public void visitTryCatchBlock(Label start, Label end, Label handler, String type) {
-      tryCatchBlocks.add(new TryCatchBlock(start, end, handler, type));
+      String exception = type == null ? null : Type.getObjectType(type).getClassName();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2583)

Reply:
I am fixing this